src/main/java/org/broadinstitute/hellbender/utils/gcs/BucketUtils.java

```diff
@@ -54,6 +54,11 @@ public static boolean isCloudStorageUrl(final String path) {
         return path.startsWith(GCS_PREFIX);
     }
 
+    public static boolean isCloudStorageUrl(final java.nio.file.Path path) {
+        // the initial "" protects us against a null scheme
+        return ("" + path.toUri().getScheme() + "://").equals(GCS_PREFIX);
```

Comment:
I respectfully disagree, the concatenation of a String with null returns the string "null" due to the language's [rules for string conversion](https://docs.oracle.com/javase/specs/jls/se8/html/jls-5.html#jls-5.1.11). I added a test for everyone's peace of mind and it confirms my understanding.